### ISSUE TITLE
best config for MultiMNIST?

### ISSUE THREAD

**Issue by duyunshu** [intel-isl/MultiObjectiveOptimization#14]
Could you please publish the best config for MultiMNIST experiments? I tried with the following but only got ~70% testing accuracy for both tasks, which is much lower than what's reported in your paper. 

```
{
    "optimizer": "SGD",
    "batch_size": 256,
    "lr": 0.0005,
    "dataset": "mnist",
    "tasks": ["L", "R"],
    "normalization_type": "none",
    "algorithm": "mgda",
    "use_approximation": true,
    "scales": {"L":0.5, "R":0.5}
}
```

Thanks!

**Comment by ozansener** [comment]
This is the hyper-parameter set we used:

optimizer=SGD|batch_size=256|lr=0.05|normalization_type=none

Note that we do not always use the final model. We perform cross-validation. We go for 100 epochs and save each model during training after each epoch. Then, we choose the model with the highest validation accuracy and test it. In other words, we do cross validation for early stopping.
